test/app/services/postServiceConfig.test.js

```diff
@@ -115,69 +203,31 @@ describe("when editing the service configuration", () => {
     updateService.mockReset();
     updateService.mockImplementation(() => Promise.resolve([]));
     getServiceById.mockReset();
-    getServiceById.mockReturnValueOnce({ ...currentServiceInfo }).mockReturnValueOnce(null);
+    getServiceById.mockReturnValueOnce({ ...currentAuthServiceInfo }).mockReturnValueOnce(null);
     res.mockResetAll();
   });
 
-  it("then it should render view with validation if service home not a valid url", async () => {
-    req.body.serviceHome = "not-a-url";
-
-    await postServiceConfig(req, res);
-    expect(res.render.mock.calls).toHaveLength(1);
-    expect(res.render.mock.calls[0][0]).toBe("services/views/serviceConfig");
-    expect(res.render.mock.calls[0][1]).toEqual({
-      authFlowType: "hybridFlow",
-      backLink: "/services/service1",
-      csrfToken: "token",
-      currentNavigation: "configuration",
-      service: {
-        ...updatedServiceModel,
-        serviceHome: req.body.serviceHome,
-      },
-      serviceId: "service1",
-      userRoles: [],
-      validationMessages: {
-        serviceHome: `${ERROR_MESSAGES.INVALID_HOME_PROTOCOL}`,
-      },
-    });
-  });
-
+  // COMMON Tests
   it("then it should render view without validation if service home url is an empty string", async () => {
+    // ARRANGE
     req.body.serviceHome = "";
 
+    // ACT
     await postServiceConfig(req, res);
+
+    // ACT
```

Comment:
Should this be ASSERT?

Reply:
It should - I got carried away typing // ACT by this point - thank you :)